ONSdigital/eq-survey-runner
# Duplicate ID detection

**_What**_
- Detect and Protect against duplicate ids in survey schema files.
- Updates parser and models to maintain register of questionnaire components and detect duplicate ids.
- Modifies a number of test cases to emulate parser behaviour when constructing models for testing.

**_How to test**_

1) Check out the branch
2) Run the unit tests
3) Run the app and verify that the app functions correctly

**_Who can test**_

Anyone except @weapdiv-david
